client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js

```diff
@@ -43,42 +62,46 @@ const ShippingZoneList = ( { site, siteId, loaded, shippingZones, isValid, trans
 				{ zonesToRender.map( renderShippingZone ) }
 			</div>
 		);
-	};
-
-	const addNewHref = loaded
-		? getLink( '/store/settings/shipping/:site/zone/new', site )
-		: '#';
+	}
 
-	const onAddNewClick = ( event ) => {
-		if ( ! loaded ) {
+	onAddNewClick = ( event ) => {
+		if ( ! this.props.loaded ) {
 			event.preventDefault();
 		}
-	};
+	}
+
+	render() {
+		const { site, siteId, loaded, isValid, translate } = this.props;
 
-	return (
-		<div>
-			<QueryShippingZones siteId={ siteId } />
-			<ExtendedHeader
-				label={ translate( 'Shipping Zones' ) }
-				description={ translate( 'These are the regions you’ll ship to. ' +
-					'You can define different shipping methods for each region. ' ) }>
-				<Button
-					href={ addNewHref }
-					onClick={ onAddNewClick }
-					disabled={ ! isValid || ! loaded }>{
-					translate( 'Add zone' ) }
-				</Button>
-			</ExtendedHeader>
-			<Card className="shipping__zones">
-				{ renderContent() }
-			</Card>
-		</div>
-	);
-};
+		const addNewHref = loaded
+			? getLink( '/store/settings/shipping/:site/zone/new', site )
+			: '#';
+
+		return (
+			<div>
+				<QueryShippingZones siteId={ siteId } />
+				<ExtendedHeader
+					label={ translate( 'Shipping Zones' ) }
+					description={ translate( 'These are the regions you’ll ship to. ' +
+						'You can define different shipping methods for each region. ' ) }>
+					<Button
+						href={ addNewHref }
+						onClick={ this.onAddNewClick }
+						disabled={ ! isValid || ! loaded }>{
+						translate( 'Add zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Add DNS Zone' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woo-bootstrap-shipping-settings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).